android/assets/jsons/Civ V - Gods & Kings/Policies.json

```diff
@@ -84,7 +84,8 @@
                 "name": "Republic",
                 "uniques": [
                     "[+1 Production] [in all cities]",
-                    "[+5]% Production when constructing [All] buildings [in all cities]"
+                    "[+5]% Production when constructing [All] buildings [in all cities]",
+                    "[+5]% Production when constructing [All] wonders [in all cities]"
```

Comment:
Is this how it was in the base game? That's our baseline here

Reply:
Yes I've double-checked with EmperorPenguin on Discord
he's definitely an experienced player in my view.

Reply:
I question this. Did he confirm in Civ 5 or in base Deciv?